ui/packages/tidb-dashboard-lib/src/apps/Statement/pages/Detail/PlanBind.tsx

```diff
@@ -42,10 +42,7 @@ const PlanBind = ({ query, plans }: PlanBindProps) => {
     }
   }, [planBindingStatus])
 
-  const hasPlanToBind = useMemo(
-    () => plans.find((plan) => !!plan.plan_digest) !== undefined,
-    [plans]
-  )
+  const hasPlanToBind = useMemo(() => plans[0].plan_can_be_bound, [plans])
```

Comment:
If the computing is not expensive, `useMemo` is unnecessary. so 

```
const hasPlanToBind = plans[0].plan_can_be_bound
```

is enough.